sass/themes/schemas/components/light/_radio.scss

```diff
@@ -55,7 +63,8 @@ $light-radio: (
             500,
         ),
     ),
-    focus-outline-color-focused: transparent,
```

Comment:
You can't remove properties just like that. If a property should be removed or renamed, a migration should be written.